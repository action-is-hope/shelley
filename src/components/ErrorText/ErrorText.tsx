/** ErrorText.tsx */
import React from "react";
import classnames from "classnames";
/* = Style API. */
import { st, classes } from "./errorText.st.css";
const ErrorText = ({
  children,
  className: classNameProp,
  ...rest
}: React.HTMLAttributes<HTMLSpanElement>) => {
  return (
    <span className={st(classnames(classes.root, classNameProp))} {...rest}>
      {children}
    </span>
  );
};

export default ErrorText;
