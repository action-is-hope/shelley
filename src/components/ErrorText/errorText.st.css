/* errorText.st.css */
@namespace "ErrorText";

.root {}
