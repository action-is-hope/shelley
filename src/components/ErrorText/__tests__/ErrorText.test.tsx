import React from "react";
import ReactDOM from "react-dom";
import ErrorText from "../ErrorText";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<ErrorText />, div);
  ReactDOM.unmountComponentAtNode(div);
});
