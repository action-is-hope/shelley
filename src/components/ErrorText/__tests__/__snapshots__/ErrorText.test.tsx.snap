// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders as a basic button with custom #className 1`] = `
<span
  className="ErrorText4259465581__root custom-class"
  data-testid="error-data-testid"
>
  Error message
</span>
`;
